File: trunk/yyandroid/src/com/yy/android/vo/ProjectCost.java
package com.yy.android.vo;

import java.io.Serializable;
import java.util.Date;

public class ProjectCost implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private int F_ID;
	private int F_ProjectID;
	private String ProjectName;
	private int F_CostType;
	private float F_CostAmount;
	private String F_OtherCost;
	private Date F_CostDate;
	private int F_OperatorID;
	private Date F_OperateTime = new Date();
	private boolean F_IsDelete;
	private boolean isChecked;
	private boolean F_IsProjectCost;
	private String CostTypeName;
	private String F_CostDescription;
	private int F_ApplyID;

	public boolean isF_IsProjectCost() {
		return F_IsProjectCost;
	}

	public void setF_IsProjectCost(boolean f_IsProjectCost) {
		F_IsProjectCost = f_IsProjectCost;
	}

	public int getF_ApplyID() {
		return F_ApplyID;
	}

	public void setF_ApplyID(int f_ApplyID) {
		F_ApplyID = f_ApplyID;
	}

	public String getF_CostDescription() {
		return F_CostDescription;
	}

	public void setF_CostDescription(String f_CostDescription) {
		F_CostDescription = f_CostDescription;
	}

	public String getCostTypeName() {
		return CostTypeName;
	}

	public void setCostTypeName(String costTypeName) {
		CostTypeName = costTypeName;
	}

	public String getProjectName() {
		return ProjectName;
	}

	public void setProjectName(String projectName) {
		ProjectName = projectName;
	}

	public boolean isChecked() {
		return isChecked;
	}

	public void setChecked(boolean isChecked) {
		this.isChecked = isChecked;
	}

	public int getF_ID() {
		return F_ID;
	}

	public void setF_ID(int in_F_ID) {
		F_ID = in_F_ID;
	}

	public int getF_ProjectID() {
		return F_ProjectID;
	}

	public void setF_ProjectID(int in_F_ProjectID) {
		F_ProjectID = in_F_ProjectID;
	}

	public int getF_CostType() {
		return F_CostType;
	}

	public void setF_CostType(int in_F_CostType) {
		F_CostType = in_F_CostType;
	}

	public float getF_CostAmount() {
		return F_CostAmount;
	}

	public void setF_CostAmount(float in_F_CostAmount) {
		F_CostAmount = in_F_CostAmount;
	}

	public String getF_OtherCost() {
		return F_OtherCost;
	}

	public void setF_OtherCost(String in_F_OtherCost) {
		F_OtherCost = in_F_OtherCost;
	}

	public Date getF_CostDate() {
		return F_CostDate;
	}

	public void setF_CostDate(Date in_F_CostDate) {
		F_CostDate = in_F_CostDate;
	}

	public int getF_OperatorID() {
		return F_OperatorID;
	}

	public void setF_OperatorID(int in_F_OperatorID) {
		F_OperatorID = in_F_OperatorID;
	}

	public Date getF_OperateTime() {
		return F_OperateTime;
	}

	public void setF_OperateTime(Date in_F_OperateTime) {
		F_OperateTime = in_F_OperateTime;
	}

	public boolean isF_IsDelete() {
		return F_IsDelete;
	}

	public void setF_IsDelete(boolean in_F_IsDelete) {
		F_IsDelete = in_F_IsDelete;
	}

}

File: trunk/CSClient/Library/Library.Model/Entriy/ProjectCost.cs
/**  版本信息模板在安装目录下，可自行修改。
* ProjectCost.cs
*
* 功 能： N/A
* 类 名： ProjectCost
*
* Ver    变更日期             负责人  变更内容
* ───────────────────────────────────
* V0.01  2015/6/26 0:16:32   N/A    初版
*
* Copyright (c) 2012 Maticsoft Corporation. All rights reserved.
*┌──────────────────────────────────┐
*│　此技术信息为本公司机密信息，未经本公司书面同意禁止向第三方披露．　│
*│　版权所有：动软卓越（北京）科技有限公司　　　　　　　　　　　　　　│
*└──────────────────────────────────┘
*/
using System;
namespace Library.Model
{
	/// <summary>
	/// ProjectCost:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class ProjectCost
	{
		public ProjectCost()
		{}
		#region Model
		private int _f_id;
		private int _f_projectid;
		private bool _f_isprojectcost= true;
		private int _f_costtype;
		private decimal _f_costamount;
		private string _f_othercost;
		private DateTime _f_costdate;
		private int _f_operatorid;
		private DateTime _f_operatetime;
		private bool _f_isdelete;
		private string _f_costdescription;
		private int? _f_applyid;
		/// <summary>
		/// 
		/// </summary>
		public int F_ID
		{
			set{ _f_id=value;}
			get{return _f_id;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int F_ProjectID
		{
			set{ _f_projectid=value;}
			get{return _f_projectid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public bool F_IsProjectCost
		{
			set{ _f_isprojectcost=value;}
			get{return _f_isprojectcost;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int F_CostType
		{
			set{ _f_costtype=value;}
			get{return _f_costtype;}
		}
		/// <summary>
		/// 
		/// </summary>
		public decimal F_CostAmount
		{
			set{ _f_costamount=value;}
			get{return _f_costamount;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string F_OtherCost
		{
			set{ _f_othercost=value;}
			get{return _f_othercost;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime F_CostDate
		{
			set{ _f_costdate=value;}
			get{return _f_costdate;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int F_OperatorID
		{
			set{ _f_operatorid=value;}
			get{return _f_operatorid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime F_OperateTime
		{
			set{ _f_operatetime=value;}
			get{return _f_operatetime;}
		}
		/// <summary>
		/// 
		/// </summary>
		public bool F_IsDelete
		{
			set{ _f_isdelete=value;}
			get{return _f_isdelete;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string F_CostDescription
		{
			set{ _f_costdescription=value;}
			get{return _f_costdescription;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? F_ApplyID
		{
			set{ _f_applyid=value;}
			get{return _f_applyid;}
		}
		#endregion Model

	}
}

